### ISSUE TITLE
Typos in AMP docstrings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2344]
|<img alt="scmikes's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| scmikes reported|
|-|-|
|Trac ID|trac#2344|
|Type|defect|
|Created|2007-01-02 06:26:41Z|

glyph,

Here are a couple of typos in the amp docstring.  You have probably read them so many times, that the mind blurs.

hope these are helpful,
Mike

------------------------------------------------------

s/whilch/which/

Later, when you want to actually do a sum, the following expression will return
a Deferred whilch will fire with the result::

===================================================
s/propogation/propagation/

You can also define the propogation of specific errors in AMP.  For example,

=======================================
s/signalled/signaled/

    - If an implementation becomes confused and starts emitting corrupt data,
      or gets keys confused with values, many common errors will be
      signalled immediately instead of delivering obviously corrupt packets.
=========================================
optional  s/braindead/brain-dead/

    - NULs will be present at regular intervals along the protocol, providing
      some padding for otherwise braindead 

=======================================================
s/argument argument/argument/

    def _wrapWithSerialization(self, aCallable, command):
        """
        Wrap aCallable with its command's argument argument de-

==================================================
s/JuiceBox/AmpBox/

class _SwitchBox(AmpBox):
    """
    Implementation detail of ProtocolSwitchCommand: I am a JuiceBox which sets
    up state for the protocol to switch.
    """

Attachments:

* [amp-doc.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/13c/13ccccf336b94f2e0c8998d2f8e5689cd13c5fe5/f55ef3a14c73c7c15c812498f6f03fa7b6ec7611.diff) (1437 bytes) - added by scmikes on 2007-01-02 06:40:52Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__2344 2344
type__defect defect
reporter__scmikes scmikes
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__amp_docstring amp docstring
time__1167719201000000 1167719201000000
changetime__1167722591000000 1167722591000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph set status to `closed`|
|-|-|

(In [19182]) Correct several docstring typos in AMP.

Author: scmikes

Reviewer: glyph

Fixes [#2344](https://github.com/twisted/twisted/issues/2344)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
